sidebars.js

```diff
@@ -7,37 +7,20 @@ module.exports = {
     },
     {
       type: "category",
-      label: "Process",
+      label: "How to Run Experiments",
       items: [
+        "process-getting-started",
         {
           type: "link",
-          label: "Nimbus Onboarding",
+          label: "Design Process (Mana)",
```

Comment:
Q: Why the explicit "(Mana)" shout-out? I don't recall seeing that in other links on the sidebar.

Reply:
🤷‍♀️ I can  remove it